editing/cefalse-boundaries-deletion.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Selecting and deleting all from the cE=true element with cE=false element at the beginning</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div contenteditable>
+  <div contenteditable="false" id="cefalse-beginning">&nbsp;</div>
+  <p id="paragraph-beginning">Lorem ipsum dolor sit amet.</p>
+</div>
+
+<div contenteditable>
+  <p id="paragraph-end">Lorem ipsum dolor sit amet.</p>
+  <div contenteditable="false" id="cefalse-end">&nbsp;</div>
+</div>
+
+<div contenteditable>
+  <div contenteditable="false" id="cefalse-boundaries-beginning">&nbsp;</div>
+  <p id="paragraph-boundaries">Lorem ipsum dolor sit amet.</p>
+  <div contenteditable="false" id="cefalse-boundaries-end">&nbsp;</div>
+</div>
+
+<script>
+    test( () => {
+      const cefalse = document.querySelector( '#cefalse-beginning' );
+      const paragraph = document.querySelector( '#paragraph-beginning' );
+      getSelection().collapse( paragraph, 0 );
+      document.execCommand( 'selectAll' );
+      document.execCommand( 'delete' );
+
+      assert_false( cefalse.isConnected, 'cE=false element was removed' );
```

Comment:
nit: Although I don't know whether there are some discussion about WPT test messages. For making the message clearer, I recommend to use "should" or "shouldn't" in the messages. For example, "cE=false element should be removed".

Reply:
Sounds reasonable, I've updated the messages.